src/inbox.rs

```diff
@@ -493,7 +493,7 @@ impl<A> From<Box<dyn MessageEnvelope<Actor = A>>> for SentMessage<A> {
 }
 
 /// An error returned in case the mailbox of an actor is full.
-pub struct MailboxFull<A>(pub Arc<Spinlock<WaitingSender<A>>>);
+pub struct MailboxFull<A>(pub Arc<Spinlock<WaitingSender<SentMessage<A>>>>);
```

Comment:
Not a blocker for this merge or the release of 0.6, but it'd be really nice to be able to replace this with a rendezvous send

Reply:
Making this generic is a step towords that, is it not?

Reply:
I agree, I was just mentioning that we must remember to make the switch to a oneshot at some point

Reply:
I recorded it as an issue: https://github.com/Restioson/xtra/issues/168